### PR TITLE
test_: Code Migration from status-cli-tests for 1 on 1 message

### PR DESCRIPTION
**Pull request by shashankshampi**
Changes:

Added 1 on 1 message Test 

Migrated contract request test from the old repository https://github.com/status-im/status-cli-tests
Added validation with signal
Added response validation
Helper and methods to support migration

### CODE REVIEW

File: tests-functional/tests/test_one_to_one_messages.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                validator.validate_response(
+                    expected_chat_id=receiving_node_pubkey,
+                    expected_message=message_text,
+                    expected_sender=f"{sending_node.name}_user"
+                )
+            else:
+                logging.error(f"Response for message {message_text} was empty or invalid")
+
+            try:
+                receiving_node.wait_for_signal("messages.new", None)
```
Comment (fbarbu15) [inline tests-functional/tests/test_one_to_one_messages.py]: same as with the old PR, we need to check that the message_text or message_id that we send in the sending_node is present in the signal

File: tests-functional/validators/message_validator.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+
+logger = logging.getLogger(__name__)
```
Comment (fbarbu15) [inline tests-functional/validators/message_validator.py]: again not using the custom logger

**Review by fbarbu15 — CHANGES_REQUESTED** [review]

File: tests-functional/tests/test_one_to_one_messages.py (hunk 1 above)
Comment (shashankshampi) [reply tests-functional/tests/test_one_to_one_messages.py]: added validation for text, name and id

File: tests-functional/validators/message_validator.py (hunk 2 above)
Comment (shashankshampi) [reply tests-functional/validators/message_validator.py]: updated to custom logger

**Review by shashankshampi — COMMENTED** [review]
Addressed all PR review.
Merging into main PR

**Pull request merged**